plugins/circleci/.toolkitrc.yml

```diff
@@ -1,5 +1,6 @@
 installs:
-  CircleCi: './lib/circleci-config'
+  CircleCi:
+    entryPoint: './lib/circleci-config'
```

Comment:
**suggestion(non-blocking)**: should we add the managed files as part of this PR?

```suggestion
    entryPoint: './lib/circleci-config'
    managedFiles:
      - '.circleci/config.yml'
```

Reply:
oops, meant to do that but forgot!